bucket/metastore.json

```diff
@@ -26,6 +26,10 @@
             "32bit": {
                 "url": "https://github.com/rasa/metastore/releases/download/v$version/metastore.exe"
             }
+        },
+        "hash": {
+            "url": "https://github.com/rasa/metastore/releases/latest",
```

Comment:
Use specific tag release for supporting downgrading with `scoop install metascore@<version>`